tests/components/deconz/test_light.py

```diff
@@ -1327,6 +1327,7 @@ async def test_non_color_light_reports_color(
     # device won't have neither color temp nor color
     with pytest.raises(AssertionError):
         assert hass.states.get("light.group").attributes.get(ATTR_COLOR_TEMP) is None
+    with pytest.raises(AssertionError):
         assert hass.states.get("light.group").attributes.get(ATTR_HS_COLOR) is None
```

Comment:
Hello @Kane610, can you look as this test? This was not caught because the second assert did raise and with both in the same `pytest.raises` context, the test passes if either of both statements raises.

Reply:
Altered the test https://github.com/home-assistant/core/pull/114021

Reply:
Continue the good work!